app/views/institutions/_profile_complaints.html.erb

```diff
@@ -105,30 +105,52 @@
-    <p><a href="https://www.ftc.gov/news-events/press-releases/2016/01/ftc-brings-enforcement-action-against-devry-university" target="_blank">Federal Trade Commission Filed Suit for Deceptive Advertising</a></p>
-                        <p><a href="https://studentaid.ed.gov/sa/sites/default/files/devry-limitation-notice.pdf" target="_blank">Notice of Intent Issued by Department of Education (ED)</a></p>
-
-        <p><a href="http://www.benefits.va.gov/gibill/comparison_tool/about_this_tool.asp#caution" target="_blank">Suspended for 85/15 violation – Flight Program</a></p>
-
-    <p><a href="http://www.benefits.va.gov/gibill/comparison_tool/about_this_tool.asp#caution" target="_blank">Denial of Recertification Application to Participate in the Federal Student Financial Assistance Programs</a></p>
-
-  <p><a href="http://www.benefits.va.gov/gibill/comparison_tool/about_this_tool.asp#ISAA" target="_blank">Iowa SAA approval ends 9/18/16 (Potential lapse in program approval and payment of GI Bill benefits.)</a></p>
-
-  <p><a href="http://www.benefits.va.gov/gibill/comparison_tool/about_this_tool.asp#ACICS" target="_blank">Accreditor under review</a></p>
-
-
-        </br><p> To learn more about caution flags, visit the <a href="http://www.benefits.va.gov/gibill/comparison_tool/about_this_tool.asp#caution" target="_blank">About this Tool</a> page.</p>
-</div></div>
+    <p>
+      These are indicators VA has determined potential students should pay attention to and consider before
+      enrolling in a program of education. A caution flag means VA or other federal agencies like the Department
+      of Education or Department of Defense have applied increased regulatory or legal scrutiny to a program of
+      education. VA will display other categories of caution flags in future releases of the GI Bill Comparison
+      Tool.
+    </p></br>
+    <p><a href="http://www.justice.gov/opa/pr/profit-college-company-pay-955-million-settle-claims-illegal-recruiting-consumer-fraud-and" target="_blank">
+      Settlement with U.S. Government
+    </a></p>
+    <p><a href="https://studentaid.ed.gov/sa/about/data-center/school/hcm" target="_blank">
+      Heightened Cash Monitoring
+    </a></p>
+    <p><a href="http://ope.ed.gov/accreditation/" target="_blank">
+      Accreditation
+    </a></p>
+    <p><a href="https://www.dodmou.com/Home/Faq" target="_blank">
+      DoD Probation For Military Tuition Assistance
+    </a></p>
+    <p><a href="https://www.ftc.gov/news-events/press-releases/2016/01/ftc-brings-enforcement-action-against-devry-university" target="_blank">
+      Federal Trade Commission Filed Suit for Deceptive Advertising
+    </a></p>
+    <p><a href="https://studentaid.ed.gov/sa/sites/default/files/devry-limitation-notice.pdf" target="_blank">
+      Notice of Intent Issued by Department of Education (ED)
```

Comment:
Per question [here](https://github.com/department-of-veterans-affairs/gi-bill-comparison-tool/issues/448#issuecomment-268828661) we're still waiting on on an anchor tag and will need to change this text to "Settlement reached with the Federal Trade Commission (FTC)". Might as well get the anchor tag first, however, before making the text change so let's wait on that from @ericks24 first.